crates/ln-dlc-node/src/node/storage.rs

```diff
@@ -124,7 +124,7 @@ impl Storage for InMemoryStore {
     ) -> Result<()> {
         let mut payments = self.payments.lock();
         match payments.get_mut(payment_hash) {
```

Comment:
🤔 If we don't need it mutable, I guess we can also change that to a simple `get`?

Reply:
We are still mutating it, it's just that the returned type is already `&mut`, so the `mut` keyword is superfluous.